p2p/server.go

```diff
@@ -404,10 +410,13 @@ func (srv *Server) Start() (err error) {
 	}
 
 	dynPeers := (srv.MaxPeers + 1) / 2
```

Comment:
what is `dynPeers` used for?

Reply:
number of peers connected through dynamic dials. normal geth client connects half of MaxPeers through dialing and fills the rest with incoming connections.

Reply:
done